src/index.js

```diff
@@ -1,11 +1,39 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import App from './components/App';
+import combineReducers from './reducers/index';
+
+const state = {
+  books: [
+    {
+      id: Math.random(),
+      title: 'Biology Conceptions',
+      category: 'Learning',
+    },
+
+    {
+      id: Math.random(),
+      title: 'Microverse Instructions',
+      category: 'Learning',
+    },
+    {
+      id: Math.random(),
+      title: 'Microverse Instructions',
+      category: 'Learning',
+    },
+  ],
+};
+
+const store = createStore(combineReducers, state);
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Provider store={store}>
+      <App />
+    </Provider>
   </React.StrictMode>,
```

Comment:
- [ ] According to the project's instructions, this should wrap `App` in` Provider` and assign the whole thing into a variable. e.g:

```javascript
const wrappedApp = (
  <Provider store={store}>
    <App />
  </Provider>
);
```